cfgov/v1/models/base.py

```diff
@@ -301,15 +301,14 @@ def serve_post(self, request, *args, **kwargs):
 
     @property
     def status_string(self):
-        page = CFGOVPage.objects.get(id=self.id)
-        if page.expired:
+        if self.expired:
             return _("expired")
-        elif page.approved_schedule:
+        elif self.approved_schedule:
             return _("scheduled")
-        elif page.live and page.shared:
-            if page.has_unpublished_changes:
-                if page.has_unshared_changes:
-                    for revision in page.revisions.order_by('-created_at', '-id'):
+        elif self.live and self.shared:
+            if self.has_unpublished_changes:
+                if self.has_unshared_changes:
+                    for revision in self.revisions.order_by('-created_at', '-id'):
```

Comment:
@richaagarwal that might be possible, I'm a bit hesitant to mess with this because I don't quite understand the way that shared state and `self.has_unshared_changes` interact.